releasenotes/notes/Added-random-circuit-from-graph-95c22eeabdea89d0.yaml

```
@@ -0,0 +1,71 @@
+---
+features_circuits:
+  - |
+    The function :func:`~qiskit.circuit.random.utils.random_circuit_from_graph` generates a random circuit that 
```

```suggestion
    Added a function :func:`~qiskit.circuit.random.utils.random_circuit_from_graph` that generates a random circuit that 
```
More in line with how we document new functionality in the release notes

Also, please update `qiskit/circuit/__init__.py` to include the function in the api docs, and make sure the link here works when you render locally the release notes (with `tox -e docs`)

---

The changes are in [ebe1de0](https://github.com/Qiskit/qiskit/pull/12474/commits/ebe1de0072977e4543dac8927e2802f599818201)
I have followed exactly what was mentioned in "Generating random circuits" section of `qiskit/circuit/__init__.py`